examples/date-picker.amp.html

```diff
@@ -31,87 +31,62 @@
   }
```

Comment:
When I tried to run this, it wouldn't load because the `amp-date-picker` experiment was off (but I did turn it on from https://www.ampproject.org/docs/reference/experimental). I noticed that several examples in the codebase have a snippet like this at the top of the `<body>` tag. Do you need to add one here or is there a different way of making this work?

```
  <script>
    setTimeout(function() {
        if (!AMP.isExperimentOn('amp-date-picker')) {
          AMP.toggleExperiment('amp-date-picker');
          location.reload();
        }
    }, 1000);
  </script>
```

Reply:
in `amphtml/examples/` I only see this snippet in the `visual-tests` folder. I think I'll keep it out of these examples to stay consistent